### PR TITLE
Separates ansible-deployment and running test builds in CI

### PR DESCRIPTION
**Pull request by navidshaikh**
To ensure we execute different operations separately and have control over what is failing.

### CODE REVIEW

**Comment by navidshaikh** [comment]
Deployment only CI failure logs : https://ci.centos.org/job/centos-container-pipeline-service-ci-pr/1493/console

**Comment by navidshaikh** [comment]
cccp-index job failure only logs: https://ci.centos.org/job/centos-container-pipeline-service-ci-pr/1494/console

**Comment by navidshaikh** [comment]
test project build failure only logs: https://ci.centos.org/job/centos-container-pipeline-service-ci-pr/1495/console

File: ci/lib.py [hunk 1]
```diff
@@ -428,6 +430,70 @@ def setup(nodes, options):
     }
 
 
+def run_cccp_index_job(jenkins_master):
+    """Run cccp-index job configured in Jenkins
+
+    Running this job on jenkins populates projects in jenkins.
```
Comment (dharmit) [inline ci/lib.py]: Running this job on jenkins "master" populates...

File: container_pipeline/workers/linter.py [hunk 2]
```diff
@@ -33,6 +33,7 @@ def handle_job(self, job):
         """
         This menthod handles the job received for dockerfile lint worker.
         """
+        print 1/0
```
Comment (dharmit) [inline container_pipeline/workers/linter.py]: What does this do? Wouldn't execution fail here with `ZeroDivisionError`? Rest of the function won't be executed IIUC.

Comment (navidshaikh) [reply container_pipeline/workers/linter.py]: Again, you should see the commit message as why this is added.
This was added intentionally to see how console output is if something goes wrong.

**Comment by navidshaikh** [comment]
Functional test failure console output https://ci.centos.org/job/centos-container-pipeline-service-ci-pr/1496/console

**Comment by navidshaikh** [comment]
> Minor stuff. LGTM otherwise.

@dharmit : addressed your review.

**Pull request merged**